__tests__/app.test.js

```diff
@@ -40,18 +40,38 @@ describe('app', () => {
       return request(app)
         .get('/api/articles/1')
         .expect(200)
-        .then(({ body: article }) => {
-          expect.objectContaining({
-            article: {
+        .then(({ body: { article } }) => {
+          expect(article).toEqual(
+            expect.objectContaining({
               author: expect.any(String),
               title: expect.any(String),
               article_id: expect.any(Number),
               body: expect.any(String),
               topic: expect.any(String),
-              created_at: expect.any(Number),
+              created_at: expect.any(String),
               votes: expect.any(Number),
-            },
-          });
+            })
+          );
+        });
+    });
+    test('Status 200 - should respond with specified article object with comment_count property', () => {
+      return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body: { article } }) => {
+          console.log(article);
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              body: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(String),
+            })
```

Comment:
we know what article should come back no? can we be more specific with our test?

Reply:
Yes! We know the exact comment count within the test data, so that should be specified. I'll update that, commit, push and merge the branch. Thanks for your help today, David.